spec/models/tag_prompt_deployment_spec.rb

```diff
@@ -1,10 +1,69 @@
 describe TagPromptDeployment do
 	let(:tag_dep) { TagPromptDeployment.new id: 1, tag_prompt: tp, tag_prompt_id: 1, question_type: "Criterion", answer_length_threshold: 5 }
+	let(:tag_dep1) { TagPromptDeployment.new id: 1, tag_prompt: tp, tag_prompt_id: 1, question_type: "Criterion", answer_length_threshold: nil, assignment_id: 1, assignment: assignment , questionnaire: questionaire}
```

Comment:
`id` for this should not be 1 right? As it's already being used by `tag_dep` in the previous line.

Reply:
I don't think this will cause any issue as they are being used in a different method.